spec/requests/api/v1/docs/display_ads_spec.rb

```diff
@@ -176,7 +176,7 @@
             body_markdown: { type: :string, description: "The text (in markdown) of the ad (required)" },
             approved: { type: :boolean, description: "Ad must be both published and approved to be in rotation" },
             published: { type: :boolean, description: "Ad must be both published and approved to be in rotation" },
-            organization_id: { type: :int32, description: "Identifies the organization to which the ad belongs" },
+            organization_id: { type: :integer, description: "Identifies the organization to which the ad belongs" },
```

Comment:
Small fix from other endpoint that came up on the Swagger [online editor](https://editor.swagger.io/)